server/services/utils/formValidation.ts

```diff
@@ -231,7 +231,10 @@ export function validateGroup({
       {
         formResponse: getIn(licence, ['curfew', 'curfewAddressReview']),
         formType: 'curfewAddressReview',
-        pageConfig: curfewConfig.curfewAddressReview,
+        pageConfig:
+          licence.curfew?.curfewAddressReview?.version === '1'
+            ? curfewConfig.curfewAddressReview
```

Comment:
curfewAddressReview on its own refers to version 1

Reply:
Is it not possible to call it `curfewAddressReviewVersion1` ?